hummingbird/ml/operator_converters/onnxml_linear.py

```diff
@@ -75,4 +75,36 @@ def convert_onnx_linear_model(operator, device=None, extra_config={}):
     )
 
 
+def convert_onnx_linear_regression_model(operator, device, extra_config):
```

Comment:
Actually I don't think you need this. The convert linear model should work.

Reply:
mmmm...that will  take a small bit of reworking on parsing; let me try.